### PR TITLE
feat(plasma-b2c): delete unnecessary dependencies/devDependencies

### PR DESCRIPTION
**Pull request by Yakutoc**
Ряд компонентов стали общими и переехали в plasma-hope, а с ними и переехала часть зависимостей, которые теперь **не нужны** в plasma-b2c.   

Что нужно удалить: 

- "@testing-library/react-hooks": "7.0.2",
- "@types/jest": "27.0.2",
- "jest": "27.2.5", 
- "ts-jest": "27.0.6",
- "react-file-drop": "3.1.2",
- "react-sortable-hoc": "2.0.0"
- "react-draggable": "4.4.3",
- jest.config.js 

:baby_chick: Download canary assets:

[colors--canary.392.4361515840.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.392.4361515840.xml)  
[color_metro_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.392.4361515840.swift)  
[color_metro_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.392.4361515840.kt)  
[color_metro_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.392.4361515840.ts)  
[color_sberHealth_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.392.4361515840.swift)  
[color_sberHealth_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.392.4361515840.kt)  
[color_sberHealth_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.392.4361515840.ts)  
[color_sbermarket_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.392.4361515840.swift)  
[color_sbermarket_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.392.4361515840.kt)  
[color_sbermarket_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.392.4361515840.ts)  
[color_sberprime_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.392.4361515840.swift)  
[color_sberprime_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.392.4361515840.kt)  
[color_sberprime_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.392.4361515840.ts)  
[color_selgros_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.392.4361515840.swift)  
[color_selgros_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.392.4361515840.kt)  
[color_selgros_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.392.4361515840.ts)  
[color_smbusiness_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.392.4361515840.swift)  
[color_smbusiness_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.392.4361515840.kt)  
[color_smbusiness_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.392.4361515840.ts)  
[PlasmaTokensColor--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.392.4361515840.swift)  
[shadow_sbermarket_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.392.4361515840.ts)  
[typo_mage_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.392.4361515840.swift)  
[typo_mage_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.392.4361515840.kt)  
[typo_mage_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.392.4361515840.ts)  
[typo_plasma_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.392.4361515840.swift)  
[typo_plasma_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.392.4361515840.kt)  
[typo_plasma_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.392.4361515840.ts)  
[typo_ruler_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.392.4361515840.swift)  
[typo_ruler_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.392.4361515840.kt)  
[typo_ruler_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.392.4361515840.ts)  
[typo_sage_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.392.4361515840.swift)  
[typo_sage_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.392.4361515840.kt)  
[typo_sage_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.392.4361515840.ts)  
[typo_sbermarket_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.392.4361515840.swift)  
[typo_sbermarket_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.392.4361515840.kt)  
[typo_sbermarket_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.392.4361515840.ts)  
[typo_soulmate_ios-swift--canary.392.4361515840.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.392.4361515840.swift)  
[typo_soulmate_kotlin--canary.392.4361515840.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.392.4361515840.kt)  
[typo_soulmate_react-native--canary.392.4361515840.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.392.4361515840.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.156.0-canary.392.4361515840.0
  # or 
  yarn add @salutejs/plasma-b2c@1.156.0-canary.392.4361515840.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-392/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-392/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-392/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-392/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-392/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-392/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-392/temple-storybook/

File: packages/plasma-b2c/package.json [hunk 1]
```diff
@@ -26,23 +26,17 @@
...
         "atLeast": 99.97
     },
     "dependencies": {
         "@salutejs/plasma-core": "1.101.0",
         "@salutejs/plasma-hope": "0.14.0",
         "@salutejs/plasma-typo": "0.32.0",
-        "react-draggable": "4.4.3",
-        "react-file-drop": "3.1.2",
-        "react-sortable-hoc": "2.0.0"
+        "react-draggable": "4.4.3"
```
Comment (Yeti-or) [inline packages/plasma-b2c/package.json:39]: все ещё нужен?

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-b2c/package.json (hunk 1 above)
Comment (Yakutoc) [reply packages/plasma-b2c/package.json:39]: @Yeti-or 

Спасибо что напомнил про `react-draggable`. Он действительно **не нужен** больше в b2c.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-392/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-392/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-392/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-392/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-392/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-392/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-392/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-392/

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-392/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-392/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-392/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-392/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-392/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-392/temple-storybook/

**Pull request merged**